src/rotp/lang/fr/labels.txt

```diff
@@ -1888,6 +1888,22 @@ SETTINGS_MOD_SCRAP_REFUND_ALL_DESC      | Option classique. Toute la ferraille s
 SETTINGS_MOD_SCRAP_REFUND_EMPIRE_DESC   | Seuls les vaisseaux en orbite sur les mondes de l'empire seront valorisés.
 SETTINGS_MOD_SHIP_SPACE_FACTOR_DESC     | Contrôle l'espace disponible dans les vaisseaux.
 
+SETTINGS_MOD_RACE_STATUS_LOG_HELP        | Les graphiques de statut Empire peuvent être affichés sur une échelle linéaire ou logarithmique.
+SETTINGS_MOD_RACE_STATUS_LOG_DESC        | Les graphiques d'état Empire peuvent être affichés sur une échelle linéaire ou logarithmique.
+SETTINGS_MOD_RACE_STATUS_LOG             | Log graphique de statut Empire : %1
+SETTINGS_MOD_RACE_STATUS_LOG_YES_DESC    | Échelle logarithmique
+SETTINGS_MOD_RACE_STATUS_LOG_NO_DESC     | Échelle linéaire
+SETTINGS_MOD_RACE_STATUS_LOG_YES_UI      | Échelle logarithmique
+SETTINGS_MOD_RACE_STATUS_LOG_NO_UI       | Échelle linéaire
+SETTINGS_MOD_RACE_STATUS_VIEW_HELP       | Les valeurs d'état de l'Empire peuvent être affichées comme absolues ou relatives.<br> Par rapport au total connu ou au joueur.
```

Comment:
Not sure where this one is displayed, so the translation may be off.

Reply:
Utilisé pour le mode de visionnement des graphiques de l'onglet "Status" du menu "Empires"

Reply:
Certes, mais où? Je vois "Échelle linéaire" mais pas "Les valeurs d'état de l'Empire peuvent être affichées comme absolues ou relatives.<br> Par rapport au total connu ou au joueur."

Reply:
L'aide (F1) n'affiche pas ça.

Reply:
Oups! J'avais oublié d'ajouter ce paramètre dans le paneau Options. Sa valeur n'était par conséquent pas sauvegardée (pas trop grave), et l'affichage du "Guide" inaccessible. C'est maintenant corrigé.
Je me suis rajouté une tâche, non urgente, de mettre à jour le "F1" local. La fonctionnalité de ces boutons n'étant pas si mystérieuse...